docs/content/Configuration/Config.mdx

```diff
@@ -314,6 +315,31 @@ module.exports = {
 };
 ```
 
+### <--{"id" : "Options Reference"}--> canSwitchSqlUser
+
+Used in [SQL API][ref-sql-api], and can be an `async` function. Default
```

Comment:
`[ref-sql-api]` isn't linked properly

Reply:
Hm, as I can see, it's defined at the end of `Config.mdx`
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/572096/180049158-2930e35b-cf66-4814-8da0-819114a3c5bd.png">

Reply:
Weird, when I tried it in the Netlify Deploy Preview the link didn't seem to work

Reply:
Just checked, the correct URL should be `/backend/sql` (instead of `/sql-api`)

Reply:
✅